### PR TITLE
[BUGFIX] Fix usage stats events

### PR DESCRIPTION
**Pull request by anthonyburdi**
Changes proposed in this pull request:
- Fixes an issue that was throwing a KeyError when running `UsageStatisticsHandler.emit`, preventing events from being sent
- Adds a test that ensures common messages are sent by checking for errors in the log
- There will be further PRs with additional tests, see GREAT-440 for a complete listing

### Definition of Done
Please delete options that are not relevant.

- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [x] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

File: great_expectations/core/usage_statistics/usage_statistics.py [hunk 1]
```diff
@@ -187,7 +189,7 @@ def build_envelope(self, message):
             + "Z"
         )
 
-        event_duration_property_name: str = f'{message["event_name"]}.duration'.replace(
+        event_duration_property_name: str = f'{message["event"]}.duration'.replace(
```
Comment (alexsherstinsky) [inline great_expectations/core/usage_statistics/usage_statistics.py:192]: @anthonyburdi I am sure you are right -- it just seems a little surprising that it has worked with our "standard" tests before (with the key being "event_name") -- are you sure about this change?

**Review by alexsherstinsky — CHANGES_REQUESTED** [review]
Just one question there...

File: great_expectations/core/usage_statistics/usage_statistics.py (hunk 1 above)
Comment (anthonyburdi) [reply great_expectations/core/usage_statistics/usage_statistics.py:192]: I thought so too, but this was a recent addition in https://github.com/great-expectations/great_expectations/pull/3729 - we have used `event_name` as a parameter in many places but the key has always been `event`.

Comment (alexsherstinsky) [reply great_expectations/core/usage_statistics/usage_statistics.py:192]: @anthonyburdi So maybe that was the issue, meaning that while the schema and message tests worked, the events were never actually emitted?..  Is that what is happening here?

**Review by alexsherstinsky — APPROVED** [review]
LGTM

**Pull request merged**